lblod/app-gelinkt-notuleren
# Consume mandatees from LMB

### Overview
This implements the LDES consumer which takes mandatees created on LMB and adds them to GN, In the corresponding public and private graphs

##### connected issues and PRs:
GN-4980


### Setup
You will need to replace the extra headers env variable, ask me for the auth token.
Also is probably your dev setup of virtuoso doesn't have enough ram, if you are running into out of memory problems increase it

### How to test/reproduce
Consume the data, check that now you have mandatees in http://mu.semte.ch/graphs/lmb-data-public, you should be able to find these mandatees in the frontend if you go to the corresponding administrative units

### Challenges/uncertainties
Waiting for a Niels answer related to in which graphs should be put de data



### Checks PR readiness
